README.md

```diff
@@ -224,6 +225,17 @@ Please note that PyTorch uses shared memory to share data between processes, so
 for multithreaded data loaders) the default shared memory segment size that container runs with is not enough, and you
 should increase shared memory size either with `--ipc=host` or `--shm-size` command line options to `nvidia-docker run`.
 
+### Building the Documentation
+
+To build documentation in various formats, you will need Sphinx.
+
+```
+conda install sphinx sphinx_rtd_theme
```

Comment:
this should be:

```
cd docs/
pip install -r requirements.txt
```

Reply:
Huh, totally did not see that that exists. Fixing.